### DISCUSSION TITLE
When will Alipay payment or WeChat Pay be supported for GitHub copilot?

### DISCUSSION THREAD

**Discussion by dwczhixingheyi** [community/community#47718] [Copilot Conversations]
### Select Topic Area

Question

### Body

I am a Chinese software developer， When will Alipay payment or WeChat Pay be supported for GitHub copilot?

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @dwczhixingheyi! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Reply by cbarney813** [reply]
Good way to keep topics on point and organized

**Comment by toyanghong** [comment]
> 宝支付或者微信支

信用卡就可以了  一直在用。

**Reply by Dxig** [reply]
国内银行卡可以吗

**Reply by cqaoi** [reply]
银联+Visa 双币

**Comment by dwczhixingheyi** [comment]
不可以。国内银行卡 不支持。
















在 2023-03-11 08:12:43，"Radeon" ***@***.***> 写道：

国内银行卡可以吗

**Comment by huanGG** [comment]
可以用国内银行卡开通paypal，然后就可以用paypal支付了

**Reply by guge2** [reply]
我用的国内银行卡（建行）配合PayPal使用，开通境外支付，然后5月1日支付失败，失败之后没看到主动发起付款的选项，只能等下一次扣费，下一次扣费是在一年后，后来我改成国内的信用卡，还是没有自动扣费。。。。

**Reply by huanGG** [reply]
我这边开通copilot也没有主动扣费，甚至上个月的账单都没了，只能看到这个月的账单。

**Comment by dwczhixingheyi** [comment]
请问以下提示是什么意思，该怎么处理？
“我们会永久限制您的账户。
经审查，我们决定永久限制您的账户，您无法再使用PayPal。”
















在 2023-03-20 17:21:11，"huanGG" ***@***.***> 写道：

可以用国内银行卡开通paypal，然后就可以用paypal支持了

**Reply by huanGG** [reply]
看不到具体信息，可以找客服申诉一下。

**Comment by haxcod** [comment]
Response #10317

**Comment by dakshithadissanayaka** [comment]
At this time, there is no official announcement from GitHub regarding the integration of Alipay or WeChat Pay as payment options for GitHub Copilot. Currently, GitHub Copilot supports payment through credit cards and PayPal.

However, there is growing demand for Alipay and WeChat Pay support, especially among developers in China. These two payment methods are widely used in China, and their integration would make GitHub Copilot more accessible to Chinese developers.

It is possible that GitHub may consider adding Alipay and WeChat Pay support in the future, but there is no specific timeline for this. In the meantime, Chinese developers can use credit cards or PayPal to pay for GitHub Copilot.

Here are some reasons why GitHub may consider adding Alipay and WeChat Pay support:

* **Increased market reach:** By supporting Alipay and WeChat Pay, GitHub could expand its reach into the Chinese market, which is one of the largest and fastest-growing markets for software development.
* **Improved user experience:** For Chinese developers, using Alipay and WeChat Pay to pay for GitHub Copilot would be a more convenient and familiar experience.
* **Increased revenue:** By making GitHub Copilot more accessible to Chinese developers, GitHub could potentially increase its revenue from subscriptions.

If you are interested in seeing Alipay and WeChat Pay support added to GitHub Copilot, you can contact GitHub support and let them know. The more users who request these payment options, the more likely it is that GitHub will consider adding them.

**Reply by mituldas1999** [reply]
Done

**Comment by mituldas1999** [comment]
Here are some reasons why GitHub may consider adding Alipay and WeChat Pay support:

Increased market reach: By supporting Alipay and WeChat Pay, GitHub could expand its reach into the Chinese market, which is one of the largest and fastest-growing markets for software development.
Improved user experience: For Chinese developers, using Alipay and WeChat Pay to pay for GitHub Copilot would be a more convenient and familiar experience.
Increased revenue: By making GitHub Copilot more accessible to Chinese developers, GitHub could potentially increase its revenue from subscriptions.

**Comment by mituldas1999** [comment]
> Here are some reasons why GitHub may consider adding Alipay and WeChat Pay support:
> 
> Increased market reach: By supporting Alipay and WeChat Pay, GitHub could expand its reach into the Chinese market, which is one of the largest and fastest-growing markets for software development. Improved user experience: For Chinese developers, using Alipay and WeChat Pay to pay for GitHub Copilot would be a more convenient and familiar experience. Increased revenue: By making GitHub Copilot more accessible to Chinese developers, GitHub could potentially increase its revenue from subscriptions.

**Comment by ogbonnad07-arch** [comment]
Copilot do you know how many billions you’re missing out on .. china is tech .. CHINA   IS.   TECH …… Bring Alipay … WE NEED THIS shii … now … like right now

**Comment by cnlnr** [comment]
If WeChat Pay is enabled, the technical costs will be recouped within a few days.
Users only need to select a plan, generate a QR code, and scan it to complete the payment.

**Comment by 54145a** [comment]
Alipay is very similar to PayPal, supporting Alipay should be easy.
